src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddress.java

```diff
@@ -11,7 +11,10 @@
 /**
  * Example: NAD+SLA+A2442389:902++DR J SMITH'
  * <br>
+ * Example: NAD+SLA+++Haematology'
+ * <br>
  * Example: NAD+SLA+++ST JAMES?'S UNIVERSITY HOSPITAL'
+ * <br>
```

Comment:
This change doesn't clarify anything for me. But if we could give some short explanation of the two different formats that could be very useful.